p2p/agent.go

```diff
@@ -143,6 +144,7 @@ var DefaultConfig = Config{
 	EnableRateLimit:   true,
 	PrivateNetworkPSK: "",
 	MaxPeers:          30,
+	MaxMessageSize:    p2p.DefaultConfig.MaxMessageSize,
```

Comment:
I think the `12MB` should be the default parameter for the iotex-core instead of the p2p.